src/coreclr/debug/daccess/request.cpp

```diff
@@ -2113,7 +2113,9 @@ ClrDataAccess::GetMethodTableData(CLRDATA_ADDRESS mt, struct DacpMethodTableData
             _ASSERTE(MTData->dwAttrClass == mtDataLocal.dwAttrClass);
             _ASSERTE(MTData->bContainsPointers == mtDataLocal.bContainsPointers);
             _ASSERTE(MTData->bIsShared == mtDataLocal.bIsShared);
-            _ASSERTE(MTData->bIsDynamic == mtDataLocal.bIsDynamic);
+            _ASSERTE(MTData->bIsDynamic == mtDataLocal.bIsDynamic
+                // DAC returns the flag value instead of 0 or 1
```

Comment:
Searching for `bIsDynamic` in SOS usage only showed places where this value is used as a boolean. I didn't any usage in CLRMD. My best guess is either this was arbitrary behavior nothing depended on or something long ago depended on it but has since been removed. Changing to 0/1 seems safe.

Reply:
I don't know what the intent was.  This code has been around a long time.  SOS uses it as a proper BOOL so changing it won't break SOS. CLRMD doesn't seem to actually check it.  Not sure about any other possible customers.